google/cloud/forseti/services/inventory/base/cloudasset.py

```diff
@@ -135,15 +129,38 @@ def load_cloudasset_data(engine, config):
                                                content_type))
 
         for future in concurrent.futures.as_completed(futures):
-            gcs_object = future.result()
-            try:
-                assets = _stream_gcs_to_database(gcs_object,
-                                                 engine,
-                                                 storage_client)
-                imported_assets += assets
-            except StreamError as e:
-                LOGGER.error('Error streaming data from GCS to Database: %s', e)
-                return _clear_cai_data(engine)
+            yield future.result()
+
+
+def load_cloudasset_data(engine, config):
+    """Export asset data from Cloud Asset API and load into storage.
+
+    Args:
+        engine (object): Database engine.
+        config (InventoryConfig): Inventory configuration on server.
+
+    Returns:
+        int: The count of assets imported into the database, or None if there
+            is an error.
+    """
+    cai_gcs_dump_paths = config.get_cai_dump_file_paths()
+
+    storage_client = storage.StorageClient()
+    imported_assets = 0
+
+    if '' in cai_gcs_dump_paths:
```

Comment:
I think the preferred way to check for empty list is:
```
if not cai_gcs_dump_paths:
   blah blah blah
```

Reply:
Updated, forgot to update this part when making the change according to andrew's comment.